plumbum.py

```diff
@@ -178,6 +184,7 @@ def list_kinesis_applications(region, filter_by_kwargs):
     'asg': list_autoscaling_group,
     'sqs': list_sqs,
     'kinesisapp': list_kinesis_applications,
+    'dynamodb': list_dynamodb
```

Comment:
Not a big deal, but in Python, I add trailing commas so future commits don't have to touch this line.